fix: [settings]set sidebar visible no response

Lighto-Ku opened the pull request:
the setting backend will be move to another thread, but it will failed while last thread destroyed. So move the SettingBackend to the main thread before call the setBackend() func.

Log: fix settings issue
Bug: https://pms.uniontech.com/bug-view-203247.html

Lighto-Ku committed:
fix: [settings]set sidebar visible no response

the setting backend will be move to another thread, but it will failed while last thread destroyed. So move the SettingBackend to the main thread before call the setBackend() func.

Log: fix settings issue
Bug: https://pms.uniontech.com/bug-view-203247.html

Clauszy reviewed (changes requested):
SettingBackend::instance() 是全局变量。他的工作线程应该在初始化时就已经决定。外面是不应该去修改的它工作线程，这不合理。且切线程会带来崩溃风险，这个变量在不同的线程都有使用到。
如果非要切，应该放到初始化时，统一管理。

Lighto-Ku committed:
fix: [workspace]foucs on title bar after batch rename

the rename bar don't have parent cause foucs on the title bar while it set invisiable.

Log: fix display issue
Bug: https://pms.uniontech.com/bug-view-203591.html

Clauszy reviewed (commented):
+1

Lighto-Ku commented:
/merge
